src/lib/tls/tls13_pqc/info.txt

```diff
@@ -0,0 +1,20 @@
+<defines>
+TLS_13_PQC -> 20230919
+</defines>
+
+<module_info>
+name -> "TLS 1.3 (PQC)"
+brief -> "Hybrid Key Exchange for TLS 1.3 with Post-Quantum Algorithms"
+</module_info>
+
+<header:public>
+</header:public>
+
+<header:internal>
+hybrid_public_key.h
+kex_to_kem_adapter.h
+</header:internal>
+
+<requires>
+tls13
```

Comment:
Should this also require `kyber`?  From a technical perspective maybe the code compiles without it, but you won't get something useful without it.

Reply:
I think it shouldn't. It's currently just an artifact of Kyber being the only available algorithm. Once #3679 is ready, we could also provide FrodoKEM-based hybrid instantiation. Consequently, we'd need to then depend on Kyber and Frodo, which doesn't really make sense to me.

Reply:
OK good point, fine to leave it like this.